PyTorch/SpeechSynthesis/FastPitch/fastpitch/model.py

```diff
@@ -242,7 +250,7 @@ def binarize_attention_parallel(self, attn, in_lens, out_lens):
     def forward(self, inputs, use_gt_pitch=True, pace=1.0, max_duration=75):
 
         (inputs, input_lens, mel_tgt, mel_lens, pitch_dense, energy_dense,
-         speaker, attn_prior, audiopaths) = inputs
+         speaker, attn_prior, audiopaths, condition) = inputs
```

Comment:
as a side-note I am wondring if we should make inputs/outputs some enum or datatype that also doesn't rely on indices to get different things out